anvil-python/anvil/commands/utils.py

```diff
@@ -49,3 +52,107 @@ def juju_login(ctx: click.Context) -> None:
     run_plan([JujuLoginStep(deployment.juju_account)], console)
 
     console.print("Juju re-login complete.")
+
+
+def validate_ssh_import(
+    ctx: click.core.Context, param: click.core.Option, value: str | None
+) -> str | None:
+    if value and re.fullmatch("(lp|gh):.+", value) is None:
+        raise click.BadParameter(
+            "--ssh-import must be of the form 'lp:username' or 'gh:username'",
+            ctx,
+        )
+    return value
+
+
+@click.command(
+    cls=FormatEpilogCommand,
+    epilog="""
+    \b
+    Create a MAAS admin account with the following details:
+    Username: admin
+    Password: VerySecure9000
+    Email: admin@company.com
+    Launchpad account to import SSH key from: lp-username
+    \b
+    maas-anvil create-admin --username admin --password VerySecure9000 --email admin@company.com --ssh-import lp:lp-username""",
+)
+@click.option(
+    "--username",
+    help="The username for the new admin account",
+    required=True,
+)
+@click.option(
+    "--password",
+    help="The password for the new admin account",
+    required=True,
+)
+@click.option(
+    "--email",
+    help="The email address for the new admin account",
+    required=True,
+)
+@click.option(
+    "--ssh-import",
+    help="Import SSH keys from Launchpad (lp:user-id) or GitHub (gh:user-id)",
+    callback=validate_ssh_import,
+)
+def create_admin(
+    username: str,
+    password: str,
+    email: str,
+    ssh_import: str | None,
+) -> None:
+    """Creates a MAAS admin account."""
+    cmd = [
+        "juju",
+        "run",
+        "maas-region/0",
+        "create-admin",
+        f"username={username}",
+        f"password={password}",
+        f"email={email}",
+    ]
+    if ssh_import:
+        cmd.append(f"ssh-import={ssh_import}")
+    try:
+        subprocess.run(cmd, check=True, capture_output=True)
+    except subprocess.CalledProcessError as e:
+        raise RuntimeError(
+            f"Failed to create MAAS admin account: {e.stderr.decode()}"
+        )
+    console.print("MAAS admin account has been successfully created.")
+
+
+@click.command(
+    cls=FormatEpilogCommand,
+    epilog="""
+    \b
+    Get a MAAS API key for the admin account:
+    maas-anvil get-api-key --username admin""",
+)
+@click.option(
+    "--username",
+    help="The username to retrieve an API key for",
+    required=True,
+)
+def get_api_key(
+    username: str,
+) -> None:
+    """Retrieves an API key for MAAS"""
+    cmd = [
+        "juju",
+        "run",
+        "maas-region/0",
+        "get-api-key",
+        f"username={username}",
+        "--format=json",
+    ]
+    try:
+        result = subprocess.run(cmd, check=True, capture_output=True)
+    except subprocess.CalledProcessError as e:
+        raise RuntimeError(f"Failed to retrieve API key: {e.stderr.decode()}")
+    api_key = json.loads(result.stdout.decode())["maas-region/0"]["results"][
+        "api-key"
+    ]
+    console.print(f"API key: {api_key}")
```

Comment:
It will be very nice to introduce a formatting option here so that we can return a structured output that can be consumed by any automation that needs the token for configuring MAAS. Tabular, YAML, JSON or something similar. To have a similar experience with existing commands, I would say let's pick the mechanism of the cluster add command which includes the token in the output